docs/serializer-formatter.rst

```diff
@@ -83,25 +83,29 @@ not well formed.
     import kafka.consumer.ConsumerIterator;
     import kafka.consumer.KafkaStream;
     import kafka.javaapi.consumer.ConsumerConnector;
-    import java.util.Properties;
+    import io.confluent.kafka.serializers.KafkaAvroDecoder;
+    import kafka.message.MessageAndMetadata;
+    import kafka.utils.VerifiableProperties;
+    import org.apache.kafka.common.errors.SerializationException;
+    import java.util.*;
 
     Properties props = new Properties();
     props.put("zookeeper.connect", "localhost:2181");
     props.put("group.id", "group1");
     props.put("schema.registry.url", "http://localhost:8081");
 
     Map<String, Integer> topicCountMap = new HashMap<String, Integer>();
```

Comment:
Minor - I suppose we're in java 7 now, so `new HashMap<>();` on the right side should be fine